assets/tests/components/nav/topNav.test.tsx

```diff
@@ -68,47 +68,55 @@ describe("TopNav", () => {
       )
 
       expect(
-        result.queryByRole("button", { name: "User Info" })
+        screen.queryByRole("button", { name: "User Info" })
       ).toBeInTheDocument()
     })
 
-    test("brings up an element with 'logged in as' text when clicked", async () => {
+    test("brings up an element with 'logged in as' text and a logout ling when clicked", async () => {
       jest.mocked(getTestGroups).mockReturnValue([TestGroups.KeycloakSso])
 
       const dispatch = jest.fn()
       const user = userEvent.setup()
-      const result = render(
+      render(
         <StateDispatchProvider state={initialState} dispatch={dispatch}>
           <BrowserRouter>
             <TopNav />
           </BrowserRouter>
         </StateDispatchProvider>
       )
 
-      expect(result.queryByText("Logged in as")).not.toBeInTheDocument()
+      expect(screen.queryByText("Logged in as")).not.toBeInTheDocument()
 
-      await user.click(result.getByTitle("User Info"))
+      await user.click(screen.getByTitle("User Info"))
 
-      expect(result.queryByText("Logged in as")).toBeInTheDocument()
+      expect(screen.queryByText("Logged in as")).toBeInTheDocument()
+
+      const logoutLink = screen.queryByRole("link", { name: "Log out" })
+
+      expect(logoutLink).toBeInTheDocument()
+
+      expect(logoutLink?.attributes.getNamedItem("href")?.value).toBe(
+        "/auth/keycloak/logout"
```

Comment:
suggestion: could we use the testing library `toHaveAttribute` matcher here?

Reply:
```suggestion
      expect(logoutLink).toHaveAttribute("href", "/auth/keycloak/logout")
```

Reply:
Heh I figured there was a better way to do this that I was forgetting. 1326f7dd

Reply:
np! just wondering, was the removal of `expect(logoutLink).toBeVisible()` intentional?

Reply:
Oops, it was not! 0c9f91be